<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="buttonCargar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cargar Archivo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCursos">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestionar Cursos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCreditos">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Conteo de Créditos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSalir">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>250</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del Curso: Laboratorio Lenguajes Formales y de Programación
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>321</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre del Estudiante: Rubén Alejandro Ralda Mejia</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Carné del Estudiante: 202111835</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
